improver/calibration/__init__.py

```diff
@@ -266,3 +266,47 @@ def add_warning_comment(forecast: Cube) -> Cube:
             "however, no calibration has been applied."
         )
     return forecast
+
+from datetime import datetime, timedelta
+
+def get_cube_from_directory(directory, cycle_point=None, max_days_offset=None, date_format='%Y%m%dT%H%MZ'):
+    """
+    loads and merges all netCDF files in a directory
+
+    To switch on the max offset filter, both cycle_point and max_days_offset 
+    need to be provided
+    Args:
+        directory (pathlib.Path):
+            The path to the directory.
+        cycle_point (str):
+            The cycle point of the forecast, used to filter files
+        max_days_offset (int):
+            Maximum number of days before cycle_point to consider files,
+            Defined as a postive int that is subtracted from the cycle_point
+        date_format (str):
+            format of the cyclepoint and datetime in the filename, used by 
+            datetime.strptime
+
+    Returns:
+        Cube
+    """
+    files = [*map(str, directory.glob("*.nc"))]
+    if len(files) == 0:
+        # This is probably too serious - is there a quiet way to handle this?
+        raise ValueError(f"No files found in {directory}")
```

Comment:
https://docs.python.org/3/library/exceptions.html#FileNotFoundError
```suggestion
        # This is probably too serious - is there a quiet way to handle this?
        raise FileNotFoundError(f"No files found in {directory}")
```